home/serializers.py

```diff
@@ -460,3 +460,9 @@ class OrderedContentSetSerializer(BaseSerializer):
     name = NameField(read_only=True)
     pages = OrderedPagesField(read_only=True)
     profile_fields = ProfileFieldsField(read_only=True)
+
+
+# TODO: Figure out what fields to make part of this serialiser
+class WhatsAppTemplateSerializer(BaseSerializer):
+    name = NameField(read_only=True)
+    body = BodyField(read_only=True)
```

Comment:
I don't think that `body` is a field on the template, and we probably want all fields from the model here (unless they're being included in the page API instead)

Reply:
Yea that's what the TODO is for, we will update the serializer once we have this new "submit via button" thing implemented, or that's what jerith and I discussed anyway

Reply:
Actually, doesn't look like we actually need the serializer yet, which is why nothing broke with it in that state.  Removed it for now, will add back once it actually gets used